src/main/java/exceptions/ConverterException.java

```diff
@@ -0,0 +1,11 @@
+package exceptions;
+
+/**
+ * The ConverterException class is an exception that is thrown when an error occurs during a conversion process.
+ */
+public class ConverterException extends Exception {
```

Comment:
**suggestion**: subclass this error instead of relying on different text messages

I had problems with writing fuzz tests because every error is the same, even though technically they are different.